### DISCUSSION TITLE
Get InboundShipmentPlan which was created in Seller Central

### DISCUSSION THREAD

**Discussion by mulloa88** [amzn/selling-partner-api-models#3260] [General]
We are planning to upload the inbound shipment plan directly to seller central via excel sheet. Then, we'd like to use APIs to pull the results into our Warehouse Management system. Is it possible to GET the inbound shipment plan via API so we have a list of the resulting shipment IDs and destination address? We would follow this up with subsequent GET calls to get the items on each shipment ID.

If the above isn't possible, we would need to find a way to get the shipment Ids first, then do a Get Shipment (by shipment ID) to get the shipment details. 

It would be much easier to just use the shipment plan ID to get the shipment IDs if possible. Also happy to hear any other suggestions if there are better options.

**Comment by jayallu** [comment]
Hello @mulloa88 

If cost is not important then you can use the recently launched inventory ledger detail report (available through report API)
The reference_id column has the shipment_ID and the items associated to that shipment. Cost will be missing though
